User Guide 3.2/Tests/test-taker-tools.md

```diff
@@ -3,41 +3,76 @@ tags: []
 
 --> 
 
-# Test-Taker Tools
+# Test-taker Tools
 
-> A set of tools that will help the Test-Taker. The majority of them are accommodation tools that aim to improve accessibility.
+> The Test-taker Tools comprise a set of tools designed to aid the Test-taker in various ways when taking tests. The majority of them are accommodation tools which aim to improve accessibility. The Test-taker Tools for a [Test](../appendix/glossary.md#test) can be found in the [Properties panel](../appendix/glossary.md#properties-panel) on the right.
 
-The test taker tools can be configured in the right sidebar either for a whole section or on a per-item basis. To display them click on the gears on the little toolbar on each section or item.
 
-![Test-Taker Tool Configuration on a Section](../resources/backend/tests/authoring/settings/section/test-taker-tools.png)
-Test-Taker Tool Configuration on a Section
+To configure the Test-taker Tools for a specific test, follow the steps below.
+
+**1.** Select the test for which the Test-taker Tools are to be configured from the [Test Library](../appendix/glossary.md#test-library) on the left, and click on *Authoring*.
+
+Test-taker Tools can be configured either for a whole section of a test, or on a per-item basis. To configure the Tools for a whole section, on the [canvas](../appendix/glossary.md#canvas) click on the cog wheels to the right of the section in question. To configure the Tools for an individual item, click on the cog wheels to the right of the item in question.
+
+In each case, the Test-taker Tools will appear in the Properties panel on the right.
+ 
+![Test-taker Tool Configuration for a Section](../resources/backend/tests/authoring/settings/section/test-taker-tools.png)
+
+**2.** Check the boxes next to each of the Test-taker Tools you wish to activate.
+
+
+The Tools available are as follows:
+ 
+## Calculator
+This option provides the test-taker with a basic calculator.
+![Calculator](../resources/delivery/features/test-taker-tools/calculator.png)
  
 ## Answer Eliminator
-Allow the test-taker to eliminate answers in some interactions. Makes more sense when you have a long list of answer choices and a learning disability
+The Answer Eliminator allows the test-taker to eliminate answers in some interactions. This is useful if there is a long list of answer choices, and the test-taker has a learning disability.
```

Comment:
This only applies the the choice interaction I believe

Reply:
This is not just a accessibility feature, but also supports the test-takers who work through answer-elimination to get to the right answer.